include/yaramod/parser/parser_driver.h

```diff
@@ -248,7 +260,7 @@ class ParserDriver
 
 	std::vector<std::string> _deferredIncludes; ///< Paths of included files when deferred includes are turned on.
 
-	std::vector<bool> _expressionArrayStack; ///< Indicates that expression array (..., ...) represents string set
+	std::vector<ExpressionArrayType> _expressionArrayStack; ///< Indicates that expression array (..., ...) represents string set
```

Comment:
Nit: I think that the doc string was not updated

Reply:
Thanks for this. I kind of realized thanks to this that I also forgot to test that the nesting actually works so I've also added test for nesting.